### PR TITLE
API Reference article

### PR DESCRIPTION
**Pull request by valo**
Contains a list of all the available APIs and how they can be used.

### CODE REVIEW

File: src/api-reference/index.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
+# API Reference
+
+This page contains a reference of all the APIs provided by Santiment.
+
+## Authentication
+
+Some of the APIs requite to have a valid API, belonging to an account with a
```
Comment (Partysun) [inline src/api-reference/index.md]: require*

File: src/api-reference/index.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
+# API Reference
+
+This page contains a reference of all the APIs provided by Santiment.
+
+## Authentication
+
+Some of the APIs require to have a valid API, belonging to an account with a
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
Some of the APIs require to have a valid API key, belonging to an account with a
```

File: src/api-reference/index.md [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+paid subscription to access all the data. The API key can be generates on your
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
paid subscription to access all the data. The API key can be generated on your
```

File: src/api-reference/index.md [hunk 4]
```diff
@@ -0,0 +1,416 @@
...
+
+After that you need to pass the API key as an additional HTTP header
+`Authorization: Apikey <YOUR_OWN_API_KEY>`. An example how to do that using curl:
+
+```bash
+curl \
+  -X POST \
+  -H "Content-Type: application/graphql" \
+  -H "Authorization: Apikey <YOUR_OWN_API_KEY>"\
+  --data '<YOUR_OWN_QUERY>}' \
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
  --data '<YOUR_OWN_QUERY>' \
```

File: src/api-reference/index.md [hunk 5]
```diff
@@ -0,0 +1,416 @@
...
+curl \
+  -X POST \
+  -H "Content-Type: application/graphql" \
+  --data '{ transactionVolume(slug:"santiment", from:"2019-01-01T00:00:00Z", to:"2019-03-01T00:00:00Z") { datetime transactionVolume }}' \
+  https://api.santiment.net/graphql
+```
+
+## Glossary
+
+There some terms used in this document. Here is a list and discription of them:
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
There are some terms used in this document. Here is a list and discription of them:
```

File: src/api-reference/index.md [hunk 6]
```diff
@@ -0,0 +1,416 @@
...
+
+* `slug` - this is an ID we use in the API to identify projects. You can find the slug
+of the projects by using the `allProjects` API
+* `ISO 8601 format` - this is the date time format used in the API. The format is `<year>-<month>-<day>T<hour>:<minute>:<seconds>Z`. For example Jan 10th 2019 12:34:56 is `2019-01-10T12:34:56Z`
+* `API key` - this is your API for accessing the premium features in the API. See the `Authentication` section for more details
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
* `API key` - this is your API key for accessing the premium features in the API. See the `Authentication` section for more details
```

File: src/api-reference/index.md [hunk 7]
```diff
@@ -0,0 +1,416 @@
...
+| --------- | ---------------------------------------------------------------------- |
+| slug      | The slug of the project                                                |
+| from      | From which date to return the volume in ISO 8601 format                |
+| to        | Till which date to return the volume in ISO 8601 format                |
+| interval  | The intervals that should be returned. Default is `1d`, which is daily |
+
+
+The query looks like this:
+
+```js
```
Comment (tspenov) [inline src/api-reference/index.md]: you gan use `graphql` instead of `js`

File: src/api-reference/index.md [hunk 8]
```diff
@@ -0,0 +1,416 @@
...
+**Max resolution: daily**
+
+Tracks the number of unique addresses that make transfers on-chain on a daily basis.
+
+Parameters:
+
+| Parameter | Description                                                            |
+| --------- | ---------------------------------------------------------------------- |
+| slug      | The slug of the project                                                |
+| from      | From which date to return the volume in ISO 8601 format                |
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
| from      | From which date to return the active addresses in ISO 8601 format                |
```

File: src/api-reference/index.md [hunk 9]
```diff
@@ -0,0 +1,416 @@
...
+| to        | Till which date to return the volume in ISO 8601 format                |
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
| to        | Till which date to return theactive addresses in ISO 8601 format                |
```

File: src/api-reference/index.md [hunk 10]
```diff
@@ -0,0 +1,416 @@
...
+#### timeseriesData
+
+To fetch the values for a given metric, slug and time interval you can use the `timeseriesData` subquery of the `getMetrics` API.
+
+Parameters:
+
+| Parameter   | Description                                                            |
+| ----------- | ---------------------------------------------------------------------- |
+| slug        | The slug of the project                                                |
+| from        | From which date to return the volume in ISO 8601 format                |
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
| from        | From which date to return the values in ISO 8601 format                |
```

File: src/api-reference/index.md [hunk 11]
```diff
@@ -0,0 +1,416 @@
...
+| to          | Till which date to return the volume in ISO 8601 format                |
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
| to          | Till which date to return the values in ISO 8601 format                |
```

File: src/api-reference/index.md [hunk 12]
```diff
@@ -0,0 +1,416 @@
...
+| interval    | The intervals that should be returned. Default is `1d`, which is daily |
+| aggregation | The aggregation to be used when fetching data for longer intervals     |
```
Comment (tspenov) [inline src/api-reference/index.md]: Maybe add: `If aggregation is not provided - every metric has a default one that fits best`

File: src/api-reference/index.md [hunk 13]
```diff
@@ -0,0 +1,416 @@
...
+**Max resolution: 1min**
+
+Returns the number of GitHub events generated by a given project repositories. Each project has one or more repositories/organizations that are tracked. A longer description of how the activity is being tracked can be found in this article: [Tracking GitHub activity of crypto projects — introducing a better approach](https://medium.com/santiment/tracking-github-activity-of-crypto-projects-introducing-a-better-approach-9fb1af3f1c32)
+
+Parameters:
+
+| Parameter | Description                                                            |
+| --------- | ---------------------------------------------------------------------- |
+| slug      | The slug of the project                                                |
+| from      | From which date to return the volume in ISO 8601 format                |
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
| from      | From which date to return the activity in ISO 8601 format                |
```

File: src/api-reference/index.md [hunk 14]
```diff
@@ -0,0 +1,416 @@
...
+| to        | Till which date to return the volume in ISO 8601 format                |
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
| to        | Till which date to return the activity in ISO 8601 format                |
```

File: src/api-reference/index.md [hunk 15]
```diff
@@ -0,0 +1,416 @@
...
+Returns the number of GitHub events generated by a given project repositories that are connected with development actions. Each project has one or more repositories/organizations that are tracked. A longer description of how the activity is being tracked can be found in this article: [Tracking GitHub activity of crypto projects — introducing a better approach](https://medium.com/santiment/tracking-github-activity-of-crypto-projects-introducing-a-better-approach-9fb1af3f1c32)
+
+The difference with the [githubActivity](#github-activity) API is that the actions counted in this API exclude events that are considered non-development related. These are:
+
+* Comments on issues
+* Issues created and closed
+* Creating of forks
+* Comments on commits
+* People following an issue
+* Downloading releaes
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
* Downloading releases
```

File: src/api-reference/index.md [hunk 16]
```diff
@@ -0,0 +1,416 @@
...
+* Project management events
+
+This metric becomes useful if you compare projects that use the github functionality for project management and issue tracking. They are going to have larger `githubActivity`, compared to projects that use github only for code, as they will generate a lot of events related to issues and tasks.
+
+Parameters:
+
+| Parameter | Description                                                            |
+| --------- | ---------------------------------------------------------------------- |
+| slug      | The slug of the project                                                |
+| from      | From which date to return the volume in ISO 8601 format                |
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
| from      | From which date to return the activity in ISO 8601 format                |
```

File: src/api-reference/index.md [hunk 17]
```diff
@@ -0,0 +1,416 @@
...
+| to        | Till which date to return the volume in ISO 8601 format                |
```
Comment (tspenov) [inline src/api-reference/index.md]:
```suggestion
| to        | Till which date to return the activity in ISO 8601 format                |
```

File: src/api-reference/index.md [hunk 18]
```diff
@@ -0,0 +1,416 @@
...
+There is a live explorer, where you can run queries directly from the browser. The
+explorer is accessible here: [https://api.santiment.net/graphiql](https://api.santiment.net/graphiql)
+
+Here is an example of running a query and seeing the results directly in the browser:
+
+[https://api.santiment.net/graphiql?query=%7B%0A%20%20transactionVolume(slug%3A%22santiment%22%2C%20from%3A%222019-01-01T00%3A00%3A00Z%22%2C%20to%3A%222019-03-01T00%3A00%3A00Z%22)%20%7B%0A%20%20%20%20datetime%0A%20%20%20%20transactionVolume%0A%20%20%7D%0A%7D](https://api.santiment.net/graphiql?query=%7B%0A%20%20transactionVolume(slug%3A%22santiment%22%2C%20from%3A%222019-01-01T00%3A00%3A00Z%22%2C%20to%3A%222019-03-01T00%3A00%3A00Z%22)%20%7B%0A%20%20%20%20datetime%0A%20%20%20%20transactionVolume%0A%20%20%7D%0A%7D)
+
+## Running requests with `curl`
+
+You can run requests with curl by passing the GraphQL query as the body of the request. Here is an exmaple how to run the transactionVolume query:
```
Comment (ivanklimuk) [inline src/api-reference/index.md]:
```suggestion
You can run requests with curl by passing the GraphQL query as the body of the request. Here is an exmaple how to run the `transactionVolume` query:
```

File: src/api-reference/index.md (hunk 7 above)
Comment (valo) [reply src/api-reference/index.md]: There is no `graphql` support in pandoc renderer. JS was the closest I can find that colors the queries in some sane way.

**Pull request merged**